chemreg/compound/tests/factories.py

```diff
@@ -20,6 +20,15 @@ class Meta:
         model = DefinedCompoundSerializer
 
 
+class DefinedCompoundFactory_V2000(DjangoSerializerFactory):
```

Comment:
is there a way that I don't have to do this to get a `v2000` molfile?

Reply:
Allow passing `v2000=True` to the factory by either using:
* [a combination of `Param` and `LazyAttribute`](https://factoryboy.readthedocs.io/en/latest/reference.html#simple-parameters) **_or_**
* [`Trait`s](https://factoryboy.readthedocs.io/en/latest/reference.html#traits)

We'll want to keep v3000 as the default as this is closest to the "real" data.